hera_sim/tests/test_vis.py

```diff
@@ -24,6 +28,59 @@ def uvdata():
         },
         antpairs=[(0, 0)]
     )
+@pytest.fixture
+def uvdataJD():
+    return io.empty_uvdata(
+        nfreq=NFREQ,
+        time_per_integ=io.SEC_PER_SDAY / NTIMES,
+        ntimes=NTIMES,
+        ants={
+            0: (0, 0, 0),
+        },
+        antpairs=[(0, 0)],
+        start_jd=2458150
+    )
+
+
+
+def test_JD(uvdata, uvdataJD):
+    freqs = np.unique(uvdata.freq_array)
+
+    # put a point source in
+    point_source_pos = np.array([[0, uvdata.telescope_lat_lon_alt[0]]])
+    point_source_flux = np.array([[1.0]] * len(freqs))
+
+    viscpu1 = vis.VisCPU(
+        uvdata=uvdata,
+        sky_freqs=np.unique(uvdata.freq_array),
+        point_source_flux=point_source_flux,
+        point_source_pos=point_source_pos,
+        nside=2**4
+    ).simulate()
+
+    viscpu2 = vis.VisCPU(
+        uvdata=uvdataJD,
+        sky_freqs=np.unique(uvdataJD.freq_array),
+        point_source_flux=point_source_flux,
+        point_source_pos=point_source_pos,
+        nside=2 ** 4
+    ).simulate()
+
+    assert viscpu1.shape == viscpu2.shape
+    assert not np.allclose(viscpu1, viscpu2, atol=0.1)
+
+@pytest.fixture
+def uvdata2():
+    return io.empty_uvdata(
```

Comment:
Maybe not the best place to put this comment, but in the future this should be changed, since `io.empty_uvdata` is quite different in `master`.

Reply:
Well that's going to be fun... Thanks for letting me know.